web/modules/custom/joinup_debug/tests/src/Functional/DbLogTest.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\Tests\joinup_debug\Functional;
+
+use Drupal\Core\Database\Database;
+use Drupal\Core\StringTranslation\StringTranslationTrait;
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\Tests\node\Traits\ContentTypeCreationTrait;
+
+/**
+ * Tests the joinup_debug content update dblog entries.
+ */
+class DbLogTest extends KernelTestBase {
+
+  use ContentTypeCreationTrait {
+    createContentType as drupalCreateContentType;
+  }
+  use StringTranslationTrait;
+
+  /**
+   * Modules to enable.
+   *
+   * @var array
+   */
+  public static $modules = [
+    'dblog',
+    'field',
+    'filter',
+    'joinup_debug',
+    'node',
+    'system',
+    'text',
+    'user',
+  ];
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+    $this->installEntitySchema('user');
+    $this->installEntitySchema('node');
+    $this->installConfig('node');
+    $this->installSchema('system', 'sequences');
+    $this->installSchema('dblog', 'watchdog');
+    $this->installSchema('node', 'node_access');
+    $this->drupalCreateContentType([
+      'type' => 'page',
+      'name' => 'Basic page',
+      'display_submitted' => FALSE,
+    ]);
+  }
+
+  /**
+   * Tests the wrong revision update.
+   */
+  public function testDbLogRevisionUpdate() {
+    /** @var \Drupal\node\NodeStorageInterface $node_storage */
```

Comment:
This is not testing the exact format of the watchdog messages, only the fact that they are being generated. I agree with this approach. I have validated manually that the message format is correct. We don't need in depth coverage since this is a temporary measure with no end user facing consequences if this regresses.